src/seahawk/seahawk_deck/thrust.py

```diff
@@ -39,28 +44,223 @@ class Thrust(Node):
     def __init__(self):
         """Initialize this node"""
         super().__init__('thrust')
+
+        self.MAX_FWD_THRUST = 36.3826715 # N
+        self.MAX_REV_THRUST = -28.6354180 # N
+
+        self.TOTAL_CURRENT_LIMIT = 70 # A
+        self.ESC_CURRENT_LIMIT = 40 # A
+
+        self.motor_positions = [
+            [0.19, 0.12, 0.047],
+            [0.19, -0.12, 0.047],
+            [-0.19, 0.12, 0.047],
+            [-0.19, -0.12, 0.047],
+            [0.105, 0.15, -0.038],
+            [0.105, -0.15, -0.038],
+            [-0.105, 0.15, -0.038],
+            [-0.105, -0.15, -0.038]
+        ]
+
+        self.motor_thrusts = [
+            [0.0, 0.0, 1.0],
+            [0.0, 0.0, 1.0],
+            [0.0, 0.0, 1.0],
+            [0.0, 0.0, 1.0],
+            [0.7071, -0.7071, 0],
+            [0.7071, 0.7071, 0],
+            [-0.7071, -0.7071, 0],
+            [-0.7071, 0.7071, 0]
+        ]
+
+        self.declare_parameter('center_of_mass_offset', [0.0, 0.0, 0.0])
+        self.center_of_mass_offset = self.get_parameter('center_of_mass_offset').value
+
+        self.add_on_set_parameters_callback(self.update_center_of_mass)
+
+        self.motor_config = self.generate_motor_config()
+        self.inverse_config = np.linalg.pinv(self.motor_config, rcond=1e-15, hermitian=False)
+
         self.subscription = self.create_subscription(Twist, 'drive/twist', self._callback, 10)
```

Comment:
The `thrust` node should subscribe to topic `desired_twist` rather than `drive/twist`. `drive/twist` is an old name for this topic and has since been updated. See [`input_xbox_one.py`](https://github.com/CabrilloRoboticsClub/cabrillo_rov_2023/blob/02662b744abcd30fe495a150c504886342bb1732/src/seahawk/seahawk_deck/input_xbox_one.py#L92C58-L92C71) on main

Reply:
Also consider changing the quote type to double quotes to be consistent with the the rest of this file and project